consensus/tree_hash/src/lib.rs

```diff
@@ -7,7 +7,7 @@ pub use merkle_hasher::{Error, MerkleHasher};
 pub use merkleize_padded::merkleize_padded;
 pub use merkleize_standard::merkleize_standard;
 
-use eth2_hashing::{hash_fixed, ZERO_HASHES, ZERO_HASHES_MAX_INDEX};
+use eth2_hashing::{hash32_concat, hash_fixed, ZERO_HASHES, ZERO_HASHES_MAX_INDEX};
```

Comment:
Please revert these changes too (will be easier to maintain with an upstream)

Reply:
04a537f5a7c593f37dcfa80493b36b363bf29daf